src/bin/sccache-dist/main.rs

```diff
@@ -453,6 +453,7 @@ fn init_logging() {
 }
 
 const MAX_PER_CORE_LOAD: f64 = 10f64;
+const SERVER_ERROR_RESET: Duration = Duration::from_secs(300);
```

Comment:
This name makes me think of how often we reset (restart) servers, which is not the intended meaning.  Could we make this `SERVER_ERROR_REMEMBER_TIMEOUT` or something?  I'm not really sure what a good name is here that's not ridiculously long.